cg/cli/workflow/balsamic/base.py

```diff
@@ -291,25 +298,26 @@ def run(context, dry, run_analysis, config_path, priority, email, case_id):
         config_path = Path.joinpath(root_dir, case_id, case_id + ".json")
 
     # Call Balsamic
-    command_str = f" run analysis" f" --account {slurm_account}" f" -s {config_path}"
+    command_str = ""
```

Comment:
Either build command string gradually or start from something. Or was there a particular reason you did it this way? To avoid split and join maybe?